content/warnings/legacy-factories.md

```diff
@@ -40,11 +40,11 @@ function render() {
 }
 ```
 
-This is an easy upgrade path if you have a lot of existing function calls.
+Eğer halihazırda çok fazla fonksiyon çağrınız varsa bu onları yükseltmek için kolay bir yoldur.
 
-## Dynamic components without JSX {#dynamic-components-without-jsx}
+## JSX Olmadan dinamik bileşenler {#dynamic-components-without-jsx}
```

Comment:
> JSX'siz dinamik bileşenler

I believe we (others) used this pattern in other translations. So it is better to stick to it.